editor/icons/ShowInFileSystem.svg

```diff
@@ -0,0 +1 @@
+<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path d="M1 1v2.25h6v4.5h3v4.5h5V10h-2.75V7.75H15V5.5H9.25V3.25H15V1z" fill="#e0e0e0"/><path d="M5.04 8.04c-1.534 0-3.475 1.169-4.173 3.423a.6.6 0 0 0 0 .33c.67 2.374 2.673 3.447 4.173 3.447s3.502-1.073 4.177-3.435a.6.6 0 0 0 0-.33C8.557 9.201 6.572 8.04 5.04 8.04zm0 1.2a2.4 2.4 0 0 1 0 4.8 2.4 2.4 0 0 1 0-4.8zm0 1.2a1.2 1.2 0 0 0 0 2.4 1.2 1.2 0 0 0 0-2.4z" fill="#5fb2ff"/></svg>
```

Comment:
You're missing an empty line at the end of the file.

Reply:
Ah okay, that is strange, I assume this is something specific to how Godot uses the icons?

Reply:
It's because your text editor didn't add it automatically I'd say, all files should have a trailing line termination

Reply:
This is the first I'm hearing of such a thing being needed. The file loads fine without it, in Godot and any other program, and I've never needed that in any other files I've ever worked with. But if it's a standard for Godot then I'll keep that in mind.

Reply:
It's a general standard tbh, especially in some environments, it ensures the file is read cleanly in many contexts, GitHub even has visual indicator for it in red, if you look at the raw file

Reply:
@Invertex This is just a code style thing. We prefer when all files have an empty line at the end (this is a bit of an anachronism from old unix days and reading from a data stream, but many projects still follow it as a default). It's not critical to using the icons specifically, but we like to keep things consistent.

Reply:
Well, for svgs optimizers default to removing all whitespace, so this never held for them.

Reply:
Yes, optimizers don't have this option most of the time and so you have to add the line manually.